reactioncommerce/reaction
# Convert login views to React

Resolves #2055 


### How to test
- Clone this branch
- Log out (if you were logged in)
- Open browser console to catch console errors (if any)
- Navigate through and use the sign in, sign up and reset password views. Ensure they work as before with the Blaze templates
- To activate social login, go to the dashboard (as an admin), go to `Login Services` and activate one/all of the options. (Credentials should be in `reaction.json` file). Log out, visit sign in/sign up and test the buttons and ensure they work as before
- On ResetPassword, (if you have email sending enabled), click link sent to your email, see the modal that pops up, ensure it works as before
- Invite a new user from the dashboard (preferably your email address) and if you have email sending enabled, click enrollment link to get redirected to set password modal. Ensure it works as before.
